### PR TITLE
Fixes GOKU-1481

### PR DESCRIPTION
**Pull request by xbonell**
Added new RichText component.

### CODE REVIEW

File: src/components/forms/RichText/index.jsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            const inlineToolbarPlugin = createInlineToolbarPlugin();
+            const { InlineToolbar } = inlineToolbarPlugin;
+            const plugins = [inlineToolbarPlugin];
+            return {
+                plugins,
+                InlineToolbar,
+            };
+        });
+
+        const useStyles = createUseStyles(styles, 'RichText');
```
Comment (dbalas) [inline src/components/forms/RichText/index.jsx:59]: this should be outside RichText function

File: src/components/utils/Tooltip/index.jsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ function Tooltip({
         overlayClassName: rootClassName,
         overlay: content,
         placement,
-        visible,
+        // visible,
```
Comment (dbalas) [inline src/components/utils/Tooltip/index.jsx:35]: is it intentional or was it for testing?

File: src/components/forms/RichText/index.jsx (hunk 1 above)
Comment (xbonell) [reply src/components/forms/RichText/index.jsx]: Fixed!

File: src/components/utils/Tooltip/index.jsx (hunk 2 above)
Comment (xbonell) [reply src/components/utils/Tooltip/index.jsx]: It's a provisional fix for non-working Tooltips

Comment (xbonell) [reply src/components/utils/Tooltip/index.jsx]: Applied @jorgandia fix.

**Pull request merged**